site/en/blog/has-m105/index.md

```diff
@@ -0,0 +1,457 @@
+---
+layout: "layouts/blog-post.njk"
+title: ":has(): the family selector"
+description: "With :has() landing in Chromium 105. Let's take a look at some of the awesome opportunities it brings to our CSS!"
+authors:
+  - jheyy
+hero: "image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/SLbZGkpDT3AG1E11MMFy.jpg"
+alt: "Image of a neon sign displaying the words Good times"
+tags:
+  - css
+date: 2022-07-25
+---
+
+Since time began (In CSS terms), we've worked with a cascade in various senses. Our styles compose a "Cascading Style Sheet". And our selectors cascade too. They can go sideways. In most cases they go downwards. But never upwards. For years, we've fantasized about a "Parent selector". And now it's finally coming! In the shape of a `:has()` pseudo selector.
+
+The `:has()` CSS pseudo-class represents an element if any of the selectors passed as parameters match at least one element.
+
+But, it's more than a "parent" selector. That's a nice way to market it. The not so appealing way might be the "conditional environment" selector. But that doesn't have quite the same ring to it. How about the "family" selector?
+
+## Browser Support
+
+Before we go any further, it's worth mentioning browser support. It's not quite there yet. But, it's getting closer. No Firefox support yet, it’s on the roadmap. But it's already in Safari and due for release in Chromium 105. All the demos in this article will tell you if they aren't supported in the browser used.
+
+{% Codepen
+  user: 'web-dot-dev'
+  id: 'RwMgZpw',
+  height: 450,
+  tab: 'result'
+%}
```

Comment:
```suggestion
{% Codepen {
  user: 'web-dot-dev',
  id: 'RwMgZpw',
  height: 450,
  tab: 'result'
} %}
```

Reply:
If you update all your CodePens to correct them like this one, the article will build. I had to check it out locally and start deleting bits to figure it out.

Reply:
Ahhh – Star! What a funny oversight! ⭐